sdk/keyvault/azure-keyvault-secrets/azure/keyvault/secrets/_models.py

```diff
@@ -300,7 +301,7 @@ def deleted_date(self):
     def recovery_id(self):
         # type: () -> str
         """
-        An identifier used to recover the deleted secret
+        An identifier used to recover the deleted secret. Will only exist if soft-delete is enabled.
```

Comment:
This implies `DeletedSecret.recovery_id` raises `AttributeError` when soft-delete is disabled but I'd guess it exists with value `None` in that case?

Reply:
Switched to saying Returns None if soft-delete is disabled.